tests/test_keras.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+
+"""Keras unit tests"""
+
+import pytest
+
+keras = pytest.importorskip('keras')
+
+from keras.models import Sequential
+from keras.layers import Activation, Conv2D, GlobalAveragePooling2D
+import numpy as np
+
+from eli5.keras import (
+    validate_doc,
+    get_activation_layer,
+    image_from_path,
+    get_target_prediction,
+)
+
+
+# We need to put this layer in a fixture object AND access it in a parametrization
+gap_layer = GlobalAveragePooling2D()
+
+@pytest.fixture(scope='module')
+def simple_seq():
+    """A simple sequential model for images."""
+    model = Sequential([
+        Activation('linear', input_shape=(32, 32, 1)), # index 0, input
+        Conv2D(10, (3, 3)),                            # index 1, conv
+        Conv2D(20, (3, 3)),                            # index 2, conv2
+        gap_layer,                                     # index 3, gap
+    ])
+    # model.summary() # TODO: print model summary if fail
```

Comment:
I think you can do ``print(model.summary())`` and this would be shown only if the test fails, because pytest does not show test stdout when they pass (not 100% sure it applies to fixtures though, if not you can print in the test).

Reply:
Perfect. Works even if I print inside a fixture.

Reply:
Would be nice to somehow display the images too. Though I'm not sure how can we automate this through Travis.

Reply:
Nice! Yeah not sure how we can show images on travis. But hopefully we should not have any cases where a test fails on travis but not locally.

Reply:
Can this TODO be removed now?

Reply:
https://github.com/TeamHG-Memex/eli5/pull/315/commits/f2260b29c362ae680404f8de784344de0df268e1